cmd/podman/compose_machine_unsupported.go

```diff
@@ -0,0 +1,14 @@
+//go:build !(amd64 || arm64)
```

Comment:
I think we can support those architectures on Linux where {docker,podman}-compose is available and can talk to the local unix socket.  My other PR attempted to do that.

Reply:
yes and I am also doing exactly this? This only here is for the machine code where we technically should never end up unless a user manually adds the machine option the the connection config file.

Reply:
Maybe I am misreading the condition.  How I understand it, we'll hit the error below when running on Linux s390, for instance.

That's why I used the `remote` build tag in the build conditions.

Reply:
We will never call this function in the first place.
`if !conf.MachineMode { return uri here } getMachineConn()`